code/datums/ai/sanity/sanityloss_behaviors.dm

```diff
@@ -1,32 +1,23 @@
-/datum/ai_behavior/say_line/insanity_murder
-	lines = list(
-				"I won't die alone...",
-				"You left me behind!",
-				"You are the problem, YOU!!",
-				"I'll destroy everything...",
-				"It will end quickly, so relax. I’ll free you from this prison we call flesh."
-				)
-
-/datum/ai_behavior/say_line/insanity_suicide
-	lines = list(
-				"It's all my fault. It’s my responsibility...",
-				"I can hear someone. It’s the sound of back home. I just can’t stop hearing it",
-				"There’s no hope left. My mind’s collapsing. Everything’s collapsing...",
-				"We will all sink and perish, devoured by madness...",
-				"There is no hope left...",
-				"It will all end, soon..."
-				)
-
-/datum/ai_behavior/say_line/insanity_wander
-	lines = list(
-				"Manager?! Manager! Open the emergency door! PLEASE LET ME OUT!!",
-				"HELP ME!!",
-				"DON'T SEND ME IN THERE! DON’T KILL ME!!",
-				"AHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHA!!",
-				"I will never get out of here..."
-				)
-
-/datum/ai_behavior/say_line/insanity_wander/perform(delta_time, datum/ai_controller/controller)
+/datum/ai_behavior/say_line/insanity_lines
+	var/init = FALSE
+	var/line_type = ""
+
+/datum/ai_behavior/say_line/insanity_lines/perform(delta_time, datum/ai_controller/controller)
+	if(!init)
+		lines = strings("insanity.json", line_type)
+		init = TRUE
+	..()
```

Comment:
To my knowledge there is no "Initializing" an ai_behavior, so I did this so that they may activate and set this once. If there is a better way, point me in the direction and I'll happily recode this. Or just tell me to copy-paste the insanity text and I'll revert this.

Reply:
New() proc should be handling this.